app/views/warehouse_reports/health/housing_status_changes/_report_download.haml

```diff
@@ -0,0 +1,20 @@
+:ruby
+  if export.present?
+    data = {
+      type: export.type,
+      query_string: export.query_string
+    }
```

Comment:
same here.  I assume there are similar ones on the HMIS side, probably worth cleaning that up.